test/test_find_library.cpp

```diff
@@ -59,15 +65,26 @@ TEST(test_find_library, find_library)
 #endif
 
   // Positive test.
-  const std::string test_lib_actual = find_library_path("test_library");
+  const std::string test_lib_actual = rcpputils::find_library_path("test_library");
   EXPECT_EQ(test_lib_actual, expected_library_path);
 
   // (Hopefully) Negative test.
-  const std::string bad_path = find_library_path(
+  const std::string bad_path = rcpputils::find_library_path(
     "this_is_a_junk_libray_name_please_dont_define_this_if_you_do_then_"
     "you_are_really_naughty");
   EXPECT_EQ(bad_path, "");
 }
 
-}  // namespace
-}  // namespace rcpputils
+TEST(test_find_library, library_path)
+{
+  // Get ground-truth values from CTest properties.
+  const auto pair = test_lib_path_and_dir();
+  const std::string expected_library_path = pair.first;
+  const std::string test_library_path = pair.second;
+
+  const std::string test_lib_actual = rcpputils::path_for_library(test_lib_dir, "test_library");
```

Comment:
This is failing to build because `test_lib_dir` is not defined.

Reply:
Ah, sorry about that. It should be fixed now.